mir-module/src/main/resources/xsl/mods-seo-meta.xsl

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xsl:stylesheet version="1.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:mods="http://www.loc.gov/mods/v3"
+                xmlns:xlink="http://www.w3.org/1999/xlink"
+                xmlns:xalan="http://xml.apache.org/xalan"
+                exclude-result-prefixes="xsl mods xlink xalan">
+
+    <xsl:include href="mods-utils.xsl"/>
+
+    <xsl:template match="mods:mods" mode="seo-meta">
+        <xsl:apply-templates select="mods:abstract|mods:abstract/@xlink:href" mode="seo"/>
+    </xsl:template>
+
+    <xsl:template match="mods:abstract[not(@altFormat)] | mods:tableOfContents[not(@altFormat)] | mods:note" mode="seo">
```

Comment:
- mods:note should not match here
- we should add a check for abstract in current lang, including fallback if abstract in current lang is not available